torch/_C/_distributed_rpc.pyi

```diff
@@ -0,0 +1,191 @@
+from typing import Tuple, Dict, Optional, List, Any, overload
+from datetime import timedelta
+import enum
+import torch
+from ._autograd import ProfilerConfig, ProfilerState
+from ._distributed_c10d import ProcessGroup, Store
+
+# This module is defined in torch/csrc/distributed/rpc/init.cpp
+
+# Defined in torch/csrc/distributed/rpc/init.cpp
+_DEFAULT_NUM_SEND_RECV_THREADS: int
+_DEFAULT_INIT_METHOD: str
+_DEFAULT_NUM_WORKER_THREADS: int
+_UNSET_RPC_TIMEOUT: float
+_DEFAULT_RPC_TIMEOUT_SEC: float
+
+class RpcBackendOptions:
```

Comment:
general question - some of the definitions here mismatched with original cpp. to name a few:
1 `rpcTimeoutSeconds` in this class is not defined 
2. `worker_id` for the init method in `WorkerInfo` is named as `id`

any reason of the discrepancy here?

Reply:
This is probably because the cpp file interface changes comparing with the time when I write the pyi file. Therefore, I need to update the python interface file to reflect the change.